themes/bootstrap3/js/common.js

```diff
@@ -668,6 +672,33 @@ function setupAutocomplete() {
   });
 }
 
+function setupCarousels(scope = document) {
```

Comment:
Would it make sense to make this a module under the VuFind namespace for better scoping? This would enable automatic initialization as part of the main VuFind.init() call and reduce the number of exports/globals you need to declare.